freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -655,33 +665,43 @@ def _create_original_to_rebuilt_nvrs_map(self):
         """
         nvrs_mapping = {}
 
-        # Get builds from all blocking advisories
-        blocking_advisories_builds = \
-            Errata().get_blocking_advisories_builds(self.event.advisory.errata_id)
-        # Get builds NVRs from the advisory attached to the message/event and
-        # then get original NVR for every build
-        for product_info in self.event.advisory.builds.values():
-            for build in product_info['builds']:
-                # Each build is a one key/value pair, and key is the build NVR
-                build_nvr = next(iter(build))
-
-                # Search for the first build that triggered the chain of rebuilds
-                # for every shipped NVR to get original NVR from it
-                original_nvr = self.get_published_original_nvr(build_nvr)
-                if original_nvr is None:
-                    continue
-                nvrs_mapping[original_nvr] = build_nvr
-                parsed_build_nvr = parse_nvr(build_nvr)
-
-                # Check builds from blocking advisories and add to the mapping
-                # all of them, that have overlapping package names
-                for block_build in blocking_advisories_builds:
-                    block_build_nvr = parse_nvr(block_build)
-                    if block_build_nvr['name'] == parsed_build_nvr['name'] and \
-                            block_build_nvr['version'] == parsed_build_nvr['version']:
-                        nvrs_mapping[block_build] = build_nvr
+        # If specific container images where requested to rebuild, process only them
+        if self.event.container_images:
```

Comment:
Same as above, the `event.container_images` are actually bundle images, and I believe here we're adding these bundle images to the nvr mapping of operator/operand images.